docs/resources/gamelift_game_server_group.md

```diff
@@ -10,7 +9,85 @@ description: |-
 
 The AWS::GameLift::GameServerGroup resource creates an Amazon GameLift (GameLift) GameServerGroup.
 
-
+## Example Usage
+
+**IMPORTANT:** Amazon GameLift Game Server Groups can take some time to finish activating. This can lead to a long-running (>25min) `terraform apply` and is normal. 
+If you encounter errors with your fleet not activating successfully, check the troubleshooting steps listed in 
```

Comment:
I believe this needs to be updated. It references a fleet but should reference a game server group and point to those troubleshooting docs

Reply:
@novekm I waffled on this, because of the downstream potential for fleet errors, but I agree. Will update.